breaking-changes.md

```diff
@@ -0,0 +1,169 @@
+# Breaking Changes
+
+## Fluture internals changed
+
+When upgrading Fluture to this version, make sure all dependencies use the
+upgraded version of Fluture as well.
+
+## Functions use simple currying
+
+Anywhere you're calling Fluture's static functions, change `, ` to `)(` or
+better yet, `) (` (see https://github.com/sanctuary-js/sanctuary/issues/438).
```

Comment:
It would be nice to use `sanctuary-js/sanctuary#438` as the link text.

Reply:
I noticed this too. I thought GitHub would do that for me.

Reply:
I believe it only works in the other direction: `<owner>/<name>#<number>` is shorthand for `[https://github.com/<owner>/<name>/issues/<number>](<owner>/<name>#<number>)`.

Reply:
* `sanctuary-js/sanctuary#438`: sanctuary-js/sanctuary#438
* `https://github.com/sanctuary-js/sanctuary/issues/438`: https://github.com/sanctuary-js/sanctuary/issues/438

Reply:
Maybe only in some contexts? :man_shrugging::point_up:

Reply:
:woman_shrugging: